src/components/Button.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import PropTypes from 'prop-types';
+
+class Button extends React.Component {
+  // eslint-disable-next-line no-useless-constructor
```

Comment:
- [ ] Kindly avoid disabling linters and fix them to make your code professional and clean.